docs/concepts/tokens/fungible-tokens/rippling.md

```diff
@@ -9,85 +7,67 @@ labels:
 
-An account can disable No Ripple on a single trust line, which can allow rippling through any pair that includes that trust line. The account can also enable rippling by default by enabling the [Default Ripple flag](#the-default-ripple-flag).
+[![Holder A sends currency through the Issuer.](/docs/img/cpt-rippling3.png "Holder A sends currency through the Issuer.")](/docs/img/cpt-rippling3.png)
 
-For example, imagine Emily has money issued by two different financial institutions, like so
+Then the Issuer side of the trust line to Holder B is reduced by 20 USD, and the account of Holder B is increased by 20 USD.
 
-[{% inline-svg file="/docs/img/noripple-03.svg" /%}](/docs/img/noripple-03.svg "Charlie --($10)-- Institution A --($1)-- Emily --($100)-- Institution B --($2)-- Daniel")
+[![Holder B receives tokens through the Issuer.](/docs/img/cpt-rippling4.png "Holder B receives tokens  through the Issuer.")](/docs/img/cpt-rippling4.png)
 
-Now Charlie can pay Daniel by rippling through Emily's address. For example, if Charlie pays Daniel $10:
+The tokens are transferred through the Issuer, but ultimately the Issuer’s balance doesn’t change. This flow of tokens is known as _rippling_. Issuing accounts must allow rippling so that their holders can transfer tokens to one another. 
 
-[{% inline-svg file="/docs/img/noripple-04.svg" /%}](/docs/img/noripple-04.svg "Charlie --($0)-- Institution A --($11)-- Emily --($90)-- Institution B --($12)-- Daniel")
+Rippling can involve several accounts as the XRP Ledger finds a path to transfer the tokens. For example, the tokens might flow between two issuer accounts that have a trust line between them.
 
-This may surprise Emily, who does not know Charlie or Daniel. Even worse, if Institution A charges her higher fees to withdraw her money than Institution B, this could cost Emily money. The No Ripple flag exists to avoid this scenario. If Emily sets it on both trust lines, then payments cannot ripple through her address using those two trust lines.
+[![Holder A transfers tokens through Issuers A and B to get to Holder B.](/docs/img/cpt-rippling5.png "Holder A transfers tokens through Issuers A and B to get to Holder B.")](/docs/img/cpt-rippling5.png)
 
-For example:
+Tokens might also flow through intermediary gateway accounts to reach a holder.
 
-[{% inline-svg file="/docs/img/noripple-05.svg" /%}](/docs/img/noripple-05.svg "Charlie --($10)-- Institution A --($1, No Ripple)-- Emily --($100, No Ripple)-- Institution B --($2)-- Daniel")
+[![Holder A transfers tokens through Issuers A and B and Gateway X to get to Holder B.](/docs/img/cpt-rippling6.png "Holder A transfers tokens through Issuers A and B and Gateway X to get to Holder B.")](/docs/img/cpt-rippling6.png)
```

Comment:
I worry that using the term "gateway" here will confuse people who come across legacy content that uses the term "gateway" differently. From 2013-2016 or so the term "gateway" was widely used in the XRPL community and in Ripple's communications to mean, essentially, "stablecoin issuer."

Perhaps "intermediary" is better here (and in the diagram) to dodge that confusion. The key thing here is that the intermediary is also just an ordinary holder except they allow rippling, not necessarily a special business or anything.

Reply:
There's a wider discussion on this - I agree that we need a term for an account that allows rippling but is not an issuer. I want to make it clear that Holders never allow rippling, Exchanges allow rippling, and Issuers allow rippling, and document their basic configurations. We can pick the term for the intermediary/exchange user after the holidays.